src/test/resources/jsonld/HT015891797

```diff
@@ -217,7 +217,7 @@
       "label" : "Quelle",
       "type" : [ "SubjectHeading" ]
     } ],
-    "label" : "Adenauer, Konrad (1876-1967) | Geschichte 1963-1967 | Quelle",
+    "label" : "Adenauer, Konrad | Geschichte 1963-1967 | Quelle",
     "type" : [ "ComplexSubject" ]
```

Comment:
Here the life dates are still missing.

Reply:
It's correct. In the `componentList` the label appears without life dates (and in the GND, too). Since the "subject chain label" should represent what's in the `componentList` the added life dates would be wrong.